platform/qt/src/qquickmapboxglrenderer.cpp

```diff
@@ -76,7 +76,19 @@ void QQuickMapboxGLRenderer::synchronize(QQuickFramebufferObject *item)
         m_map->setPitch(quickMap->pitch());
     }
 
-    if (syncStatus & QQuickMapboxGL::ColorNeedsSync && m_map->isFullyLoaded()) {
-        m_map->setPaintProperty("background", "background-color", quickMap->color());
+    if (m_map->isFullyLoaded()) {
```

Comment:
This should be style loaded, not fully loaded, otherwise we are probably going to see a color flick.